README.md

```diff
@@ -1129,6 +1129,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Samyak Jain](https://github.com/samyakjain10)
   - [Sanchit Arora](https://github.com/sanchitarorra)
   - [Sandeep Kumar Lenka](https://github.com/sandeeplenka)
+   -[Sangamesh s](https://github.com/sangu143)
```

Comment:
Incorrect link provided; only include the link to your GitHub profile!

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
```suggestion
  - [Sangamesh s](https://github.com/sangu-143)
```